ocs_ci/utility/deployment.py

```diff
@@ -0,0 +1,62 @@
+"""
+Utility functions that are used as a part of OCP or OCS deployments
+"""
+import json
+import logging
+
+import requests
+
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+def get_ocp_ga_version(channel):
+    """
+    Retrieve the latest GA version for
+
+    Args:
+        channel (str): the OCP version channel to retrieve GA version for
+
+    Returns:
+        str: latest GA version for the provided channel.
+            An empty string is returned if no version exists.
+
+    """
+    logger.debug("Retrieving GA version for channel: %s", channel)
+    url = "https://api.openshift.com/api/upgrades_info/v1/graph"
+    headers = {"Accept": "application/json"}
+    payload = {"channel": f"stable-{channel}"}
+    r = requests.get(url, headers=headers, params=payload)
+    nodes = r.json()["nodes"]
+    if nodes:
+        versions = [node["version"] for node in nodes]
+        versions.sort()
+        ga_version = versions[-1]
+        logger.debug("Found GA version: %s", ga_version)
+        return ga_version
+    logger.debug("No GA version found")
+    return ""
+
+
+@retry(AssertionError, 20, 30, 1)
+def verify_ready_machine_count():
+    """
+    Retrieve mcp and ensure that readyMachineCount == machineCount
+
+    Raises:
+        AssertionError: if the readyMachineCount != machineCount
+
+    """
+    out = run_cmd("oc get mcp -o json")
+    mcp_data = json.loads(out)
+
+    for mcp in mcp_data["items"]:
+        logger.info(
+            "Verifying readyMachineCount for MachineConfigPool %s",
+            mcp["metadata"]["name"],
+        )
+        machine_count = mcp["status"]["machineCount"]
+        ready_machine_count = mcp["status"]["readyMachineCount"]
+        assert ready_machine_count == machine_count
```

Comment:
we already have a function for this 
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L2611-L2642

Reply:
Removed this function and am now using `wait_for_machineconfigpool_status`